content/docs/getting-started.md

```diff
@@ -18,109 +18,109 @@ redirect_from:
 
-Like any unfamiliar technology, React does have a learning curve. With practice and some patience, you *will* get the hang of it.
+किसी भी अनजान  टेक्नोलॉजी की तरह React का भी एक सीखने की अवस्था है | अभ्यास और धीरज के साथ आपको *जल्द ही* React समझ में आजाएगी |
 
-### First Examples {#first-examples}
+### पहला उदहारण {#first-examples}
 
-The [React homepage](/) contains a few small React examples with a live editor. Even if you don't know anything about React yet, try changing their code and see how it affects the result.
+[React के होमपेज](/) में लाइव एडिटर के साथ React के कुछ छोटे उद्धरण हैं | अगर आप React के बारे में कुछ नहीं जानते तो कोड में कुछ बदलाव करके देखें कि वो कैसे परिणाम को बदलता है |
 
-### React for Beginners {#react-for-beginners}
+### शुरू करने वालों के लिए React {#react-for-beginners}
 
-If you feel that the React documentation goes at a faster pace than you're comfortable with, check out [this overview of React by Tania Rascia](https://www.taniarascia.com/getting-started-with-react/). It introduces the most important React concepts in a detailed, beginner-friendly way. Once you're done, give the documentation another try!
+अगर आप को लगता है की React के दस्तावेज आपकी गति पर नहीं चल रहे तो [Tania Rascia के इस ओवरव्यू को देखें](https://www.taniarascia.com/getting-started-with-react/) | ये React के सबसे ज्यादा ज़रूरी कॉन्सेप्ट्स से आपका विस्तार और शुरुवातकर्ता-अनुकूल परिचय करता है | एक बार आप इसे पूरा करलें आप दस्तावेजों को दुबारा देख सकते हैं!
 
-### React for Designers {#react-for-designers}
+### डिज़ाइनरों के लिए React {#react-for-designers}
 
-If you're coming from a design background, [these resources](https://reactfordesigners.com/) are a great place to get started.
+आगर आप डिज़ाइनर बैकग्राउंड से आते हैं, [ये साधन](https://reactfordesigners.com/) आपकी शुरुवात करने में मदद करेंगे |
 
-### JavaScript Resources {#javascript-resources}
+### जावास्क्रिप्ट कि रिसोर्सेज़ {#javascript-resources}
 
-The React documentation assumes some familiarity with programming in the JavaScript language. You don't have to be an expert, but it's harder to learn both React and JavaScript at the same time.
+React का दस्तावेज़ मानता है की आप जावास्क्रिप्ट के साथ पहले से ही सूचित हैं | आपको जावास्क्रिप्ट में माहिम होने कि ज़रूरत नहीं है, लेकिन जावास्क्रिप्ट और React को एक साथ सीखना मुश्किल हैं |
 
-We recommend going through [this JavaScript overview](https://developer.mozilla.org/en-US/docs/Web/JavaScript/A_re-introduction_to_JavaScript) to check your knowledge level. It will take you between 30 minutes and an hour but you will feel more confident learning React.
+हम अनुरोध करते है की आप [इस जावास्क्रिप्ट के ओवरव्यू को देखे](https://developer.mozilla.org/en-US/docs/Web/JavaScript/A_re-introduction_to_JavaScript) ताकि आप अपने जानकारी के स्तर को आक सकें | इसमें आपको आधे से एक घंटा लगेगा लेकिन आप React को सिखने के प्रति और अच्छ महसूस करेंगे |
```

Comment:
`आप React को सिखने के प्रति और अच्छ महसूस करेंगे` => `आप React सीखने के लिए अधिक आत्मविश्वास महसूस करेंगे`

Reply:
`के लिए अधिक` => `में अधिक`
I missed this during previous review. My bad.